Animal Competition/Java/Game.java

```diff
@@ -53,25 +18,103 @@ void startGame(){
 
       int userGameStart = userConfirmation.nextInt();
       
-    // next part of game, will work on 
+    // next part of game, currently working on
       if (userGameStart == 1) {
-        System.out.println("Select your Animal:");
+        System.out.println("Select the first Animal:(Enter the number corresponding to the animal you want.)");
```

Comment:
Did you notice that in my sample project that I only ever use `System.out` in one place?  I abstracted the output to a function that knows how to "output" text.  If in the future I wanted the output to go to a GUI component or text-to-voice, etc, I can just make the change in one small place rather than rewriting most of the program.